lib/app_profiler/viewer/middleware/firefox.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require "app_profiler/yarn/command"
+require "app_profiler/yarn/with_firefox_profiler"
+
+module AppProfiler
+  module Viewer
+    class FirefoxRemoteViewer < BaseViewer
```

Comment:
```suggestion
    class VernierRemoteViewer < BaseViewer
```

Reply:
The actual viewer is firefox, not vernier. We don't call speedscope "stackprof viewer"

Reply:
Yes, you're right. It makes it confusing now that there's multiple backends and viewers. We should probably add docs to each viewer indicating which backend it is meant for.